src/components/WorkOrder/MobileWorkingWorkOrderView.js

```diff
@@ -261,11 +271,21 @@ const MobileWorkingWorkOrderView = ({ workOrderReference }) => {
           isNoAccess ? 'closed with no access' : 'closed'
         }`
       )
+      for (let i = 0; i < localStorage.length; i++) {
+        const key = localStorage.key(i)
+        if (key && key.startsWith('closeWorkOrder')) {
+          localStorage.removeItem(key)
+        }
+      } // Clear cached form data
```

Comment:
could we move this to a clear cache method

Reply:
I gave that a go but it feels a bit premature - that could help more if we need to do something similar with the desktop work order close form also